README.md

```diff
@@ -1,5 +1,5 @@
 # GitTesting
 
 This is just a seggestion
-
+Another change. How can I make the changes visible to the author?
```

Comment:
I'm happy with this.

Reply:
Wait, don't commit yet. Going to add something more